app/src/main/res/css/modules/column.sass

```diff
@@ -39,13 +39,15 @@
     margin-top: 5rem
     font-size: 1.3rem
   .items_container
-    overflow-x: hidden
-    overflow-y: scroll
-    overflow-x: hidden
+    position: relative
+    overflow: auto
```

Comment:
You can add word-break: break-all; but it is still better to keep overflow-x hidden.